wodles/aws/aws-s3.py

```diff
@@ -402,10 +414,10 @@ def db_maintenance(self, aws_account_id, aws_region):
                     error_msg=e))
             sys.exit(10)
 
-    def marker_only_logs_after(self, aws_region, aws_account_id):
+    def marker_only_logs_after(self, aws_region, aws_account_id, only_logs_after):
```

Comment:
Why is `only_logs_after` a parameter in this function? It is an attribute in the class. I have reviewed the code and I haven't found a single place where this function is called using a different value than `self.only_logs_after`:
https://github.com/wazuh/wazuh/blob/eeb47de5006539bc8b5ee863160a5be15649f8ed/wodles/aws/aws-s3.py#L456
https://github.com/wazuh/wazuh/blob/eeb47de5006539bc8b5ee863160a5be15649f8ed/wodles/aws/aws-s3.py#L464